memflow-win32/examples/read_bench.rs

```diff
@@ -50,7 +50,7 @@ fn rwtest<T: VirtualMemory>(
                 module.base().as_u64() + module.size() as u64,
             );
 
-            while done_size < read_size {
+            while done_size < read_size * *o * (i.trailing_zeros() as usize + 1) {
```

Comment:
The second part looks a bit weird, maybe put it into a separate variable that has a clearer name?